packages/admin-stories/src/docs/icons/stories/AllIcons.stories.tsx

```diff
@@ -0,0 +1,97 @@
+import { ClearInputButton } from "@comet/admin";
+import * as icons from "@comet/admin-icons";
+import { Grid, InputAdornment, InputBase, SvgIcon, Typography } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+import styled from "styled-components";
+import { useDebounce } from "use-debounce";
+
+const matchesSearchQuery = (str: string, query: string): boolean => {
+    if (!query.length) return true;
+
+    let strIncludesPartOfSearchQuery: boolean = false;
+
+    query.split(" ").forEach((part) => {
```

Comment:
Why do you split the search string when you use `includes` for the search? You could use `includes` for the whole string as well 🤔

Reply:
I think, splitting is probably useful because it's more flexible. You don't have to know the exact name of an icon to find it.

E.g. the "RadioChecked" icon can be found by the queries "RadioChecked", "Radio Checked", "Checked Radio", ...

Reply:
My logic here was: if you're unsure if an icon is called "Date" or "Calendar", you could quickly type in half of each word to find it quickly, e.g. "cal dat".